<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseControllerBase</class>
 <widget class="QWidget" name="CourseControllerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="layout3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>210</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="courseNumber">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="courseTitle">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Q3ButtonGroup" name="controls">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="frameShape">
       <enum>Q3GroupBox::NoFrame</enum>
      </property>
      <widget class="QPushButton" name="btnSelect">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDeselect">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDeselectNodeps">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSelectNodeps">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnUseMe">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Depend</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnSelectNodeps</sender>
   <signal>clicked()</signal>
   <receiver>CourseControllerBase</receiver>
   <slot>selectionForced()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelect</sender>
   <signal>clicked()</signal>
   <receiver>CourseControllerBase</receiver>
   <slot>selectionRequested()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeselectNodeps</sender>
   <signal>clicked()</signal>
   <receiver>CourseControllerBase</receiver>
   <slot>deselectionForced()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeselect</sender>
   <signal>clicked()</signal>
   <receiver>CourseControllerBase</receiver>
   <slot>deselectionRequested()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUseMe</sender>
   <signal>clicked()</signal>
   <receiver>CourseControllerBase</receiver>
   <slot>requestPrefChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
